app/src/types/service.js

```diff
@@ -22,11 +22,5 @@ export type Service = t.TypeOf<typeof ServiceModel>;
  * @returns {Service}
  */
 export function ServicePublicToAppService(from: ServicePublic): Service {
-  return {
-    serviceId: from.serviceId,
-    serviceName: from.serviceName,
-    organizationName: from.organizationName,
-    departmentName: from.departmentName,
-    version: from.version
-  };
+  return from;
```

Comment:
`ServicePublic === Service` ? if that's the case we don't need this method anymore (and we can remove `ServicePublic`)... if that's not the case, you have to do the conversion

Reply:
`ServicePublic` is generated by the autorest tool and we cannot remove it. `Service` is needed to perform type validation using flow-runtime, so in effect we need both and we have to do the conversion, right?

Reply:
then perhaps add that comment explaining why that function is structured in that way

assume that somebody will have to work on and maintain your code, what he's going to think seeing a function that just returns its only parameter? what questions he's going to ask? (not that you may not be there to answer those questions) <--- this is a general rule